multimodal/src/autogluon/multimodal/utils/cache.py

```diff
@@ -128,6 +132,9 @@ def collate(self, x: List[Dict]):
         x
             A list of batch results. Each batch is one (nested) dictionary.
         """
+        if BBOX in x[0]:  # for detection outputs just sort the batches
+            return x
+
```

Comment:
Could we modify the return format of `mmdet_image` to avoid this task specific code?